src/dsp_tools/utils/logger_config.py

```diff
@@ -0,0 +1,45 @@
+import sys
+from datetime import datetime
+from pathlib import Path
+
+from loguru import logger
+
+logger_savepath = (Path.home() / ".dsp-tools" / "logger.log").absolute()
+
+
+def logger_config() -> None:
+    """
+    This function configures the log files.
+    Currently, there are three sinks:
+    - logger.log in the home directory for development purposes
+    - warnings.log in the cwd only with level warning and higher for the user (no stack-trace)
+    - print output on the terminal, formatted the same as the warnings.log
+    """
+    # If this is not removed, the default formatting is also printed out on the terminal
+    logger.remove()
+
+    text_format = "<level>{time:YYYY-MM:DD HH:mm:ss.SSS} | {level: <8} | {message}</level>"
+    rotation_size = "500 MB"
+    retention_number = 30
```

Comment:
This would retain 15 GB of logs, which seems too much for me. I guess that 5 x 0.5 GB = 2.5 GB would be enough.

Reply:
I looked at the current logfiles. They are 104 MB, so I changed the number now to 100 and kept the retention number. They are big enough as it is...